### DISCUSSION TITLE
Localization for settings

### DISCUSSION THREAD

**Discussion by EightEggs** [mulaRahul/keyviz#52] [Ideas]
Is there any plan for localization of settings?

**Comment by mulaRahul** [comment]
Currently, that's not in the plans but it'll be great if the community contributes the same (as I'm not much familiar with languages other than English). Machine translation is an option but I don't want to consider it because it's not always correct.

**Reply by EightEggs** [reply]
I'm willing to translate it to Chinese (and maybe Janpanese as well), but I know nothing about Dart.(qwq)
Hopefully there can be any ways to contribute?

**Reply by mulaRahul** [reply]
That's great! You don't need to know anything about **Dart** to contribute.

Hopefully in the next release, will be able to add the same. I haven't checked the localization options for flutter yet, so first I'll have to go through the same first. After that we can work on localizing the app in **Chinese** and **Japanese** if you're confident in it. Most probably the same will be done with **.json** (key-value pairs) or we can use a text file as well. 

Thanks in advance 😁!

**Comment by xingxinag** [comment]
2.0还是不支持切换语言吗？
